_sources/Selection/BooleanValuesandBooleanExpressions.rst

```diff
@@ -63,7 +63,13 @@ two values are equal to one another.
 .. activecode:: ch05_2
 
     print(5 == 5)
+
     print(5 == 6)
+    
+    j = "hel"
+    print(j + "lo" == "hello")
+
+    
 
 In the first statement, the two operands are equal, so the expression evaluates
```

Comment:
I think a sentence or two explaining the third example and emphasizing the point of the third example is that python evaluates each side of the `==` statement before it tests for equality is super important. @pearcej do you agree?

Reply:
I have reworded the active code to remove order of operation confusion and added comments about the third question below

Reply:
Yes, and I think this rewording addresses the issue.